banyand/measure/measure_suite_test.go

```diff
@@ -102,7 +105,6 @@ func setUp() (*services, func()) {
 	flags = append(flags, "--etcd-listen-client-url="+listenClientURL, "--etcd-listen-peer-url="+listenPeerURL)
 	moduleDeferFunc := test.SetupModules(
 		flags,
-		repo,
```

Comment:
If you remove `repo`, `moduleDeferFunc` can not stop it gracefully.

Reply:
If it is not removed, the unit test will block at https://github.com/apache/skywalking-banyandb/blob/1c19243df23f8350ea5c1542fd914c49e8698960/pkg/test/setup.go#L49 and cannot be terminated normally due to discovery_mock `gracefulstop` method.

log screensnapt:
![image](https://user-images.githubusercontent.com/13257127/211202607-453cb617-5702-4108-8658-a5b825d7b8b2.png)

Reply:
Do you have any advice?

Reply:
https://github.com/apache/skywalking-banyandb/pull/236#discussion_r1064253100